force-app/main/default/translations/pt_BR.translation-meta.xml

```diff
@@ -0,0 +1,8026 @@
+    </customLabels>
+    <customLabels>
+        <name>conMergePortalUser</name>
+        <label>É um usuário do Portal/da Community</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSearchPlaceholder</name>
+        <label>Pesquisar contatos</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSelectAll</name>
+        <label>Selecionar todos</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSelectContact</name>
+        <label>Selecionar contato</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSelectedContacts</name>
+        <label>Contatos selecionados</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeStageComplete</name>
+        <label>Fase completa</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeStageCurrent</name>
+        <label>Fase atual</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeStageNotStarted</name>
+        <label>Fase não iniciada</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeWinnerAsstText</name>
+        <label>Escolha esta coluna como o registro mestre</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeDRSStep1</name>
+        <label>Selecione os Conjuntos de registros duplicados que você deseja mesclar</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeFoundDRS</name>
+        <label>Duplicar conjuntos de registros</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeDRSPickBtnLabel</name>
+        <label>Escolher</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeErrorNoDRSFound</name>
+        <label>&lt;b&gt;Sem conjuntos de registros duplicados.&lt;/b&gt;&lt;/br&gt; Nenhum conjunto de registros duplicados encontrado. Verifique se as regras duplicadas estão ativas. &lt;a href=&quot;https://help.salesforce.com/articleView?id=duplicate_rules_map_of_reference.htm&amp;type=5&quot; target=&quot;_blank&quot;&gt;Mais informações&lt;/a&gt;.</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeErrorNoReadDSRObjPermission</name>
+        <label>Você precisa ter permissões de leitura em Conjuntos de registros duplicados para mesclar Contatos.</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeTooManyContactError</name>
+        <label>Muitos contatos duplicados. Use a opção Pesquisar contato para mesclar contatos</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSearchConText</name>
+        <label>Procure Contatos específicos para ver se eles têm duplicatas</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSeeDupConText</name>
+        <label>Veja todos os Contatos duplicados encontrados usando regras de correspondência. &lt;a href=&apos;https://powerofus.force.com/IP_Contacts&apos; target=&apos;_blank&apos;&gt;&lt;b&gt; Conte-me mais &lt;/b&gt;&lt;/a&gt;</label>
```

Comment:
This was changed to an html quote entity in a recent bug PR.  Unless this is changed to match, the url will be stripped.

This is quite likely to be an issue in all translations.